### ISSUE TITLE
GF test_kalapi_metrics.py & test_khaled_metrics.py

### ISSUE THREAD

**Issue by impallari** [m4rc1e/mf-glyphs-scripts#8]
Hi Mark,
/Google Fonts/test_kalapi_metrics.py and 
/Google Fonts/test_khaled_metrics.py
both ask for instances to have customParameters for the metrics (typoAscender, typoDescender, etc..)

This is no needed since they are inherited form the Masters.
You should check the masters instead of the instances.

Also, after the instances check I get this error in Kalapi's check:
```
Traceback (most recent call last):
  File "test_kalapi_metrics.py", line 167, in <module>
    main_glyphsapp()
  File "test_kalapi_metrics.py", line 140, in main_glyphsapp
    consistent_vert_instances = vert_metrics_match(font)
  File "test_kalapi_metrics.py", line 108, in vert_metrics_match
    instance_metrics[i][key] = float(instance.customParameters[key])
TypeError: float() argument must be a string or a number
```

And this one in Kalhed's check:
```
Traceback (most recent call last):
  File "test_khaled_metrics.py", line 166, in <module>
    main_glyphsapp()
  File "test_khaled_metrics.py", line 139, in main_glyphsapp
    consistent_vert_instances = vert_metrics_match(font)
  File "test_khaled_metrics.py", line 108, in vert_metrics_match
    instance_metrics[i][key] = float(instance.customParameters[key])
TypeError: float() argument must be a string or a number
```

**Comment by m4rc1e** [comment]
Hey Pablo,

I decided on including metrics for instances because all metrics should be the same for each weight.

If they differ on the masters, they will be interpolated.

I am more than happy to drop this and just test if the masters are all the same, then this will be a non issue. To work around this, there is a script. **Fixes > Copy metrics from masters to instances**

**Comment by impallari** [comment]
Yes, Metrics should be defined (and tested) in the master, not in the instances.

**Comment by m4rc1e** [comment]
Hey @impallari,

This should be fixed in be0d09c. Apologies for the delay.

Marc
